src/frontend/js/utils/test/factories.ts

```diff
@@ -0,0 +1,2 @@
+export const BlockMapFactory = '{"blocks":[{"key":"brt6g","text":"ce texte est ajouté dans code-block.\n","type":"code-block","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"fh6un","text":"","type":"unstyled","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"6an2m","text":"ce texte est en gras.","type":"unstyled","depth":0,"inlineStyleRanges":[{"offset":0,"length":27,"style":"BOLD"}],"entityRanges":[],"data":{}}],"entityMap":{}}';
```

Comment:
A factory is a function, taking arguments related to an entity and returning the wanted entity populated with the arguments you give. IMO you should change this to a real factory.